content/numerical/NumberTheoreticTransform.h

```diff
@@ -12,43 +12,39 @@
  */
```

Comment:
The doc comment still says `May return negative values.`; remove that and mention that inputs have to be in [0, mod) instead.

Update author/date as well.